unit-tests/func/MUR/l500/test-MSS-streaming.cpp

```diff
@@ -0,0 +1,43 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Please keep filenames all lower-case. It just makes things simpler.
Is "MSS" some standard term? I never heard of it, and I'm not sure it adds anything.

Reply:
Yes, MSS = Main Success Scenario. if you are not familiar with it maybe others are not also, will be replaced